core/src/test/groovy/life/qbic/domain/usermanagement/policies/PasswordEncryptionPolicySpec.groovy

```diff
@@ -82,4 +82,21 @@ class PasswordEncryptionPolicySpec extends Specification {
         password = "abcdefghihdeo"
     }
 
+    def "Equal raw user passwords must not create the same secret" () {
```

Comment:
Very nice! However I'd also suggest adding a check for the entire key length(similiar to the salt length check) to ensure the key adheres to the defined bit length 👍